src/Protocol/Diaspora/Repository/DiasporaContact.php

```diff
@@ -234,6 +234,18 @@ public function getByUrl(UriInterface $uri, ?bool $update = self::UPDATE_IF_MISS
 	 */
 	public function updateFromProbeArray(array $data): Entity\DiasporaContact
```

Comment:
Before throwing exceptions we should ensure to handle them when the function is called. Because otherwise (for example when called in a worker process), the worker will stop, be executed again, will stop, executed again, ...

Reply:
This method is called only twice, in `Model\Contact.php:1399` and `2494` and both places have now a `try catch` block.